.github/workflows/nodejs-ci.yml

```diff
@@ -216,11 +221,29 @@ jobs:
         if: ${{ !cancelled() && !contains( inputs.excluded_jobs, 'build' )  }}
         run: |
           ${{ inputs.build_command }}
+      - name: Developer Tests
+        id: developer_tests
+        if: ${{ always() && inputs.run_dev_test && github.event.pull_request.base.ref == 'qa' }}
+        run: |
+          ${{ inputs.package_manager }} run test
+      - name: Developer Test Coverage Report 
+        uses: MishaKav/jest-coverage-comment@main
+        if: ${{  always() && contains('failure,success', steps.developer_tests.outcome) }}
+        with:
+          coverage-summary-path: ./coverage/coverage-summary.json
+          title: Coverage Summary
+          badge-title: Coverage
+          hide-comment: false
+          create-new-comment: false
+          hide-summary: false
+          junitxml-title: Junit Summary
+          junitxml-path: ./coverage/report.xml     
```

Comment:
coverage-summary-path, junitxml-path must be variable inputs, it can have the above mentioned values as default.

Reply:
done